unittests/test_copy_model.py

```diff
@@ -1,9 +1,36 @@
+from django.db import models
+
 from dojo.models import Endpoint, Endpoint_Status, Engagement, Finding, Product, Test, User
 
 from .dojo_test_case import DojoTestCase
 
 
-class TestCopyFindingModel(DojoTestCase):
+class EqualityHelpers:
+    def compare_querysets(self, queryset_1, queryset_2):

```

Comment:
What about to use https://docs.djangoproject.com/en/5.1/topics/testing/tools/#django.test.TransactionTestCase.assertQuerySetEqual
I have never used that but based on the name, it might be compatible.

Reply:
Great find! This was a drop in replacement when I ran tests locally'

Thanks for the suggestion 😄